Python/sort/Sort_An_Arraysof0s1sand2s.py

```diff
@@ -0,0 +1,39 @@
+#This problems are variation of famous Dutch national flag problem.
+def sort012(arr,n):
+    listofZero = []
+    listofOne = []
+    listofTwo = []
+
+    for i in range(0,n):
+        if arr[i] == 0:
+            listofZero.append(0)
+        elif arr[i] == 1:
+            listofOne.append(1)
+        else:
+            listofTwo.append(2)
+
+    Total = listofZero+listofOne+listofTwo
+    return Total 
+
+
+
+
+arr = list(map(int,input("Enter the Number:\n").split()))
+n = len(arr)
+print(sort012(arr,n))
+
+'''
+Time Complexity is O(n)
+Space Complexity is O(1)
+
+Input: 
+N = 5
+arr[]= {0 2 1 2 0}
+
+Output:
+0 0 1 2 2
```

Comment:
Program is not matching with Sample Input & Output
**Display message is not present in Sample part**